netci.groovy

```diff
@@ -73,27 +73,28 @@ class WcfUtilities
             return 
         }
 
-        wcfRepoSyncServiceCount++
+        wcfPRServiceCount++
```

Comment:
So that's a little complicated: 

* For each run of the CI build queue generator, each queue is guaranteed to have exactly one number assigned to it. It will not vary for the lifetime of this CI build queue... 
* When the generator is run again, *these queue numbers may change* 
* However, because the ordering of the queue generation is deterministic, the generated number will be the same for a given queue unless there's a change in, say, the supported branch/OS/buildconfiguration list

Does this explanation help? (and alleviate your fears :smile:)